device-telemetry/app/com/microsoft/azure/iotsolutions/devicetelemetry/webservice/v1/controllers/AlarmsController.java

```diff
@@ -25,6 +23,7 @@
     private final IAlarms alarms;
 
     private static final int DELETE_LIMIT = 1000;
+    private static final int DEVICE_LIMIT = 1000;
```

Comment:
Is this supposed to be duplicated?

Reply:
one is number of deletes, other is max number of devices. It's possible they could be different numbers at some point